packages/cli/src/util/dev/templates/error_404.jst

```diff
@@ -6,6 +6,4 @@
     {{?}}
     <span class="devinfo-line">ID: <code>{{! it.request_id }}</code>
   </p>
-  
-  <a href="https://vercel.link/404"><div class="note">Click here to learn more about this error.</div></a>
```

Comment:
I believe this code is generated so it would probably end up here again

Reply:
I believed these were manually updated by us from time to time